.Documentation/articles/toc.yml

```
@@ -41,6 +41,8 @@
       href: innoactive-creator/default-behaviors.md
     - name: Default Conditions
       href: innoactive-creator/default-conditions.md
+    - name: Snap Zones
+      href: innoactive-creator/snapzones.md
```

```suggestion
      href: innoactive-creator/snap-zones.md
```

Update the file name, too.